### DISCUSSION TITLE
使用小技巧

### DISCUSSION THREAD

**Discussion by weekend-project-space** [weekend-project-space/top-rss-list#12] [Show and tell]
**0.添加订阅小技巧**
 - site:ruanyifeng.com 模糊匹配对应的网站
 - title:阮一峰  模糊匹配网站标题

**1.经典布局下-键盘快捷键**
- 文章列表
   left right up down 在订阅源和文章列表文章之间跳转
   enter 打开选中
- 阅读器视图
  a d 文章切换
  w s 文章滚动
  q 退出 e 内嵌视图 r 阅读视图 t 打开源网站

**2. 快速开启inbox布局**
- https://webfollow.cc/?layout=inbox

**Comment by weekend-project-space** [comment]
快速订阅rss内容小技巧

chrome安装[rss-subscription-extensio](https://chromewebstore.google.com/detail/rss-subscription-extensio/nlbjncdgjeocebhnmkbbbdekmmmcbfjd) 插件

点击订阅源[https://weekendproject.online/index.xml](https://weekendproject.online/index.xml)

进入插件订阅页面下拉选项点击管理-> RSS 订阅选项 -> 添加 （说明 webfollow.cc 网址 https://webfollow.cc/setup?s=%s） -> 选中设为默认

**Comment by psterman** [comment]
innoreader
